include/oneapi/dpl/experimental/kt/esimd_radix_sort_onesweep.h

```diff
@@ -538,25 +538,25 @@ struct __radix_sort_onesweep_submitter<KeyT, RADIX_BITS, THREAD_PER_TG, PROCESS_
                 oneapi::dpl::__ranges::__require_access(__cgh, __rng);
                 auto __data = __rng.data();
                 __cgh.depends_on(__e);
-                radix_sort_onesweep_slm_reorder_kernel<KeyT, _Output, decltype(__data), RADIX_BITS, THREAD_PER_TG, PROCESS_SIZE>
+                radix_sort_onesweep_slm_reorder_kernel<KeyT, _Output, decltype(__data), RADIX_BITS, THREAD_PER_TG, PROCESS_SIZE, IsAscending>
                     K(__n, __stage, __output, __data, __tmp_data);
                 __cgh.parallel_for<_Name...>(__nd_range, K);
             });
         }
     }
 };
 
-template <typename _ExecutionPolicy, typename KeyT, typename _Range, ::std::uint32_t RADIX_BITS,
-          bool IsAscending, ::std::uint32_t PROCESS_SIZE>
-std::enable_if_t<!::std::is_unsigned_v<KeyT>, void>
+template <typename _ExecutionPolicy, typename KeyT, typename _Range, ::std::uint32_t RADIX_BITS, bool IsAscending,
+          ::std::uint32_t PROCESS_SIZE>
+std::enable_if_t<!::std::is_unsigned_v<KeyT> && !::std::is_signed_v<KeyT>, void>
 onesweep(_ExecutionPolicy&& __exec, _Range&& __rng, ::std::size_t __n)
 {
     // TODO: remove this when the implementation below can compile for other data types
 }
```

Comment:
@dmitriy-sobolev, please see my comment below.

Reply:
According to another conversation (https://github.com/oneapi-src/oneDPL/pull/947#discussion_r1196515280), we agreed to leave it. 
However, would not it be better to leave a static_assert here to prevent compilation?

Reply:
If we insert ```static_assert```, we have compile time error for the ```float``` type.
Let's save as is.